docs/adr/0001-actors.rst

```diff
@@ -0,0 +1,110 @@
+ADR 0001 - Actors
+=================
+
+Status
+------
+
+Proposed
+
+
+Context
+-------
+
+There are different kinds of actors that need to interact with Marsha.
+
+First we have the people managing a Marsha instance.
+
+Then we have people linking their own website (this website is a "consumer site") to a Marsha instance, to host videos.
+
+These consumer sites can host many customers. We call these organizations. And these organizations have managers that can administrate some things about authors, video sharing between authors...
+
+Next we have the video authors, belonging the organizations.
+
+And finally we have the users coming to watch videos.
+
+
+Decision
+--------
+
+Let's separate those 5 actors / roles:
+
+"staff"
++++++++
+
+Purpose
+~~~~~~~
+
+To manage a Marsha instance
+
+Implementation
+~~~~~~~~~~~~~~
+
+These are simply instances of the Django ``User`` model, with the flag ``is_staff`` set to ``True``.
+
+
+"admins"
+++++++++
+
+Purpose
+~~~~~~~
+
+To manage the link between a consumer site and a Marsha instance, and the organizations of this consumer site on the instance.
```

Comment:
"the organizations of this consumer site"
-> you sound like each consumer site has its own organizations when in fact organizations are orthogonal to consumer sites right?